### PR TITLE
Add docs for network policy logging

### PR DESCRIPTION
**Pull request by manuelbuil**
Signed-off-by: Manuel Buil <mbuil@suse.com>

### CODE REVIEW

File: docs/advanced/advanced.md [hunk 1]
```diff
@@ -428,3 +428,15 @@ helm repo update
 helm install --create-namespace -n cattle-logging-system rancher-logging-crd rancher-charts/rancher-logging-crd
 helm install --create-namespace -n cattle-logging-system rancher-logging --set additionalLoggingSources.k3s.enabled=true rancher-charts/rancher-logging
 ```
+
+## Additional Network Policy Logging
+
+Dropped packets by network policies can be logged. The log is printed by iptables NFLOG action and shows the packet details and the network policy that bloked it.
```
Comment (brandond) [inline docs/advanced/advanced.md:434]:
```suggestion
Packets dropped by network policies can be logged. The packet is sent to the iptables NFLOG action, and shows the packet details, including the name of the network policy that blocked it.
```

File: docs/advanced/advanced.md [hunk 2]
```diff
@@ -428,3 +428,15 @@ helm repo update
...
+
+To do so, install ulogd2 and configure `[log1]` to read on `group=100`. Then, restart the ulogd2 service for the new config to be committed.
```
Comment (brandond) [inline docs/advanced/advanced.md:436]:
```suggestion
To convert the NFLOG packets to log entries, install ulogd2 and configure `[log1]` to read on `group=100`. Then, restart the ulogd2 service for the new config to be committed. 
```

Comment (brandond) [reply docs/advanced/advanced.md:436]: For people who don't want to install and configure ulogd2 while troubleshooting, is there an example tcpdump command that can be used to quickly print the packets?

File: docs/advanced/advanced.md [hunk 3]
```diff
@@ -428,3 +428,15 @@ helm repo update
...
+
+Dropped packets by network policies can be logged. The log is printed by iptables NFLOG action and shows the packet details and the network policy that bloked it.
+
+To do so, install ulogd2 and configure `[log1]` to read on `group=100`. Then, restart the ulogd2 service for the new config to be committed. 
+
+When traffic gets blocked, the log will appear in `/var/log/ulog/syslogemu.log`. If there is a lot of traffic, the logging file could grow very fast. To control that, set the "limit" and "limit-burst" iptables parameters approprietly by adding the following annotations:
+* kube-router.io/netpol-nflog-limit=<LIMIT-VALUE>
+* kube-router.io.io/netpol-nflog-limit-burst=<LIMIT-BURST-VALUE>
+
+in the network policy manifest. Default values are limit=10/minute and limit-burst=10. Check iptables manual for more information on these fields
```
Comment (brandond) [inline docs/advanced/advanced.md:442]:
```suggestion
When a packet is blocked by network policy rules, a log message will appear in `/var/log/ulog/syslogemu.log`. If there is a lot of traffic, the logging file could grow very fast. To control that, set the "limit" and "limit-burst" iptables parameters by adding the following annotations to the network policy in question:
* kube-router.io/netpol-nflog-limit=<LIMIT-VALUE>
* kube-router.io.io/netpol-nflog-limit-burst=<LIMIT-BURST-VALUE>

Default values are `limit=10/minute` and `limit-burst=10`. Check the iptables manual for more information on the format of and possible values for these fields.
```

**Review by brandond — CHANGES_REQUESTED** [review]
some wording nits

File: docs/advanced/advanced.md (hunk 1 above)
Comment (manuelbuil) [reply docs/advanced/advanced.md]: thanks! My English....

File: docs/advanced/advanced.md (hunk 2 above)
Comment (manuelbuil) [reply docs/advanced/advanced.md]: Good idea, just added it

**Review by rbrtbnfgl — APPROVED** [review]

**Pull request merged**